src/client/pps/util.go

```diff
@@ -69,6 +73,60 @@ func InputCommits(input *Input) []*pfs.Commit {
 				ID:   input.Cron.Commit,
 			})
 		}
+		if input.Github != nil {
+			result = append(result, &pfs.Commit{
+				Repo: &pfs.Repo{RepoNameFromGithubInfo(input.Github.URL, input.Github.Name)},
+				ID:   input.Github.Commit,
+			})
+		}
 	})
 	return result
 }
+
+// RepoNameFromGithubInfo returns the normalized repo name
+func RepoNameFromGithubInfo(url string, name string) string {
+	if name != "" {
+		return name
+	}
+	// We know URL looks like:
+	// "https://github.com/sjezewski/testgithook.git",
+	tokens := strings.Split(url, "/")
+	last := tokens[len(tokens)-1]
+	tokens = strings.Split(last, ".")
+	return tokens[0]
+}
+
+// ValidateGithubCloneURL returns an error if the provided URL is invalid
+func ValidateGithubCloneURL(url string) error {
```

Comment:
Git instead of Github. Unless this only applies to github clone urls, but I don't think that's the case.